dlt/pipeline/pipeline.py

```diff
@@ -956,6 +958,21 @@ def sql_client(self, schema_name: str = None, credentials: Any = None) -> SqlCli
         schema = self._get_schema_or_create(schema_name)
         return self._sql_job_client(schema, credentials).sql_client
 
+    def fs_client(self, schema_name: str = None, credentials: Any = None) -> FSClientBase:
```

Comment:
more or less analog to sql_client

Reply:
I'd rename it to `_fs_client` until we discuss the exact interface.